tests/util/server/src/lib.rs

```diff
@@ -304,6 +304,8 @@ async fn get_tcp_listener_stream(
   futures::stream::select_all(listeners)
 }
 
+pub const TEST_SERVERS_COUNT: usize = 28;
```

Comment:
Can we somehow compute this number so we don't find ourselves chasing it again in a few months?